students/mariusz_michalczyk/lesson_04_unit_testing/TestCollatz.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import Collatz
+
+
+class TestCollatzFunction(unittest.TestCase):
+    def test_type(self):
+        with self.assertRaises(TypeError):
+            Collatz.get_collatz_nr("aeou")
+
+    def test_expected_collatz_nr(self):
+        self.assertEqual(Collatz.get_collatz_nr(8), 4, "Collatz function returned unexpected value")
```

Comment:
That message is too verbose, unittest sucks and will only show you the offending line, but even that is enough to figure out the error. You can put more effort into naming instead, so that when the test fails, you don't even have to read the message. In that exercise it seems a bit strange, but in real code names like `test_validation_succeeds_on_missing_wheel` are a way to go. Try to make it a statement about code for example `test_collatz_returns_4_on_8`.